### PR TITLE
Add missing Overseas Care Package items

### PR DESCRIPTION
**Pull request by Latiosu**
* **Addressed Issue(s)**: https://github.com/rathena/rathena/issues/5688

* **Server Mode**: Renewal

* **Description of Pull Request**: 

Adds following items (and their item combos):
- [Bunker Shield](https://www.divine-pride.net/database/item/2184)
- [Solomon's Key](https://www.divine-pride.net/database/item/2191)
- [Miracle Blue Rose](https://www.divine-pride.net/database/item/5878)
- [Exorcist Glasses](https://www.divine-pride.net/database/item/5916)
- [Monster Fish Gills](https://www.divine-pride.net/database/item/5921)
- [Dragon Knight Eye Patch](https://www.divine-pride.net/database/item/5924)
- [Zealotus Doll](https://www.divine-pride.net/database/item/5975)
- [Antonio's Coat](https://www.divine-pride.net/database/item/15249)
- [Ixion Wing](https://www.divine-pride.net/database/item/18129)
- [Magical Booster](https://www.divine-pride.net/database/item/18720)
- [Old Pink Poo Hat](https://www.divine-pride.net/database/item/18959)
- [Catherina Von Blood](https://www.divine-pride.net/database/item/19163)
- [Guardian Processor](https://www.divine-pride.net/database/item/19170)
- [Black Veil](https://www.divine-pride.net/database/item/19460) - Illusion of Vampire bonus is commented out as placeholder
- [Diabolus Wing](https://www.divine-pride.net/database/item/19495) - Temple of the Demon God bonuses are commented out as placeholder
- [Gardener Shoes](https://www.divine-pride.net/database/item/22050)
- [Gravekeeper Shoes](https://www.divine-pride.net/database/item/22052)
- [Dragon Trainer Shoes](https://www.divine-pride.net/database/item/22058)
- [Hunter Shoes](https://www.divine-pride.net/database/item/22060)
- [Mechanic's Shoes](https://www.divine-pride.net/database/item/22062)
- [Fisherman's Shoes](https://www.divine-pride.net/database/item/22091)
- [Exorcist's Shoes](https://www.divine-pride.net/database/item/22093)
- [Gardener Shoes](https://www.divine-pride.net/database/item/22094)
- [Fallen Angel Shoes](https://www.divine-pride.net/database/item/22096)
- [Dragon Trainer Shoes](https://www.divine-pride.net/database/item/22097)
- [Mechanic Shoes](https://www.divine-pride.net/database/item/22099)
- [Spirit King's Ring](https://www.divine-pride.net/database/item/28394)
- [Yin Yang Talisman](https://www.divine-pride.net/database/item/28560)
- [Rose Bundle A](https://www.divine-pride.net/database/item/23251)
- [Orleans's Bundle A](https://www.divine-pride.net/database/item/23252)
- [Black Shiba Inu Bundle A](https://www.divine-pride.net/database/item/23253)
- [Kardui Bundle A](https://www.divine-pride.net/database/item/23255)
- [Etran Bundle A](https://www.divine-pride.net/database/item/23266)
- [Pororoca Shoes Bundle A](https://www.divine-pride.net/database/item/23294)

### CODE REVIEW

**Review by Atemo — CHANGES_REQUESTED** [review]

File: db/re/item_combo_db.txt [hunk 1]
```diff
@@ -2217,3 +2217,22 @@
...
+18720:1626,{ .@r = getequiprefinerycnt(EQI_HAND_R); .@matk = 15*.@r; if(.@r>=10){ .@matk += 50; bonus bVariableCastrate,-10; } bonus bMatkRate,.@matk; }
+18720:1665,{ .@r = getequiprefinerycnt(EQI_HAND_R); .@matk = 15*.@r; if(.@r>=10){ .@matk += 50; bonus bVariableCastrate,-10; } bonus bMatkRate,.@matk; }
+18720:28201,{ .@r = getequiprefinerycnt(EQI_HAND_R); .@dmg = 2; if(.@r>=5){ .@dmg += 3; .@def = -5; if(.@r>=7){ .@dmg += 4; .@def -= 5; } bonus2 bSubClass,Class_Normal,.@def;} bonus2 bMagicAddClass,Class_Boss,.@dmg; }
+18720:28202,{ .@r = getequiprefinerycnt(EQI_HAND_R); .@dmg = 2; if(.@r>=5){ .@dmg += 3; .@def = -5; if(.@r>=7){ .@dmg += 4; .@def -= 5; } bonus2 bSubClass,Class_Normal,.@def;} bonus2 bMagicAddClass,Class_Boss,.@dmg; }
+19170:1549,{ .@r = getequiprefinerycnt(EQI_HAND_R); .@aspd = 0; .@bonus1 = 0; .@bonus2 = 0; if (.@r >= 7) { .@aspd += 10; .@bonus1 += 30; .@bonus2 += 50; } if (.@r >= 9) { .@aspd += 10; .@bonus1 += 30; .@bonus2 += 50; } if (.@r >= 10) { bonus2 bSkillCooldown,"NC_PILEBUNKER",-1000; } bonus2 bSkillAtk,NC_BOOSTKNUCKLE,.@bonus1; bonus2 bSkillAtk,NC_VULCANARM,.@bonus2; }
```
[the remark this answers is not in the document]
Comment (Latiosu) [reply db/re/item_combo_db.txt:2227]: Oops I also forgot to apply the ASPD rate bonus here
```suggestion
19170:1549,{ .@r = getequiprefinerycnt(EQI_HAND_R); if (.@r >= 7) { .@aspd += 10; .@bonus1 += 30; .@bonus2 += 50; if (.@r >= 9) { .@aspd += 10; .@bonus1 += 30; .@bonus2 += 50; if (.@r >= 10) { bonus2 bSkillCooldown,"NC_PILEBUNKER",-1000; } } bonus2 bSkillAtk,NC_BOOSTKNUCKLE,.@bonus1; bonus2 bSkillAtk,NC_VULCANARM,.@bonus2; bonus bAspdRate,.@aspd; } }
```

File: db/re/item_combo_db.txt [hunk 2]
```diff
@@ -2217,3 +2217,25 @@
 28027:15250:19310:20906:22173:28566,{ bonus bBaseAtk,3*(min(BaseLevel,150)/10); bonus2 bAddClass,Class_All,5; bonus bAspd,1; }
 28616:15250:19310:20906:22173:28566,{ bonus bMatk,3*(min(BaseLevel,150)/10); bonus2 bMagicAtkEle,Ele_Holy,5; bonus bVariableCastrate,-10; }
 28740:15250:19310:20906:22173:28566,{ bonus bMatk,3*(min(BaseLevel,150)/10); bonus bAspdRate,5; bonus bAspd,1; }
+2191:2782,{ bonus bUseSPrate,-5; }
+15249:4243,{ bonus bDef,250; bonus2 bSubEle,Ele_All,5; bonus2 bSubEle,Ele_Dark,-5; skill "AL_TELEPORT",1; bonus4 bAutoSpellWhenHit,"AL_TELEPORT",1,-500,0; if (.@r >= 7) { bonus2 bResEff,Eff_Freeze,10000; if (.@r >= 9) { bonus bNoKnockback; } } }
```
Comment (Atemo) [inline db/re/item_combo_db.txt:2221]:
```suggestion
15249:4243,{ .@r = getequiprefinerycnt(EQI_ARMOR); bonus bDef,250; bonus2 bSubEle,Ele_All,5; bonus2 bSubEle,Ele_Dark,-5; skill "AL_TELEPORT",1; bonus4 bAutoSpellWhenHit,"AL_TELEPORT",1,-500,0; if (.@r >= 7) { bonus2 bResEff,Eff_Freeze,10000; if (.@r >= 9) { bonus bNoKnockback; } } }
```

**Review by Atemo — CHANGES_REQUESTED** [review]
Do you mind to update the SQL files too? (run yaml2sql tool)

**Comment by idk-whoami** [comment]
> Do you mind to update the SQL files too? (run yaml2sql tool)

how to use tool
i'm looking for guide

**Comment by Latiosu** [comment]
> > Do you mind to update the SQL files too? (run yaml2sql tool)
> 
> how to use tool
> i'm looking for guide

I put together a gist to help explain. Happy to continue the conversation over there 👍 
https://gist.github.com/Latiosu/ab4d2604c55f051cd4f2397718b01460

**Comment by Atemo** [comment]
Thanks !

**Pull request merged**